plugins/templates/create.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const hoek = require('hoek');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/templates',
```

Comment:
What makes a template so special that it needs to be by {id} and not by it's name and version?

There are numerous other services in the real world that follow a more RESTful pattern. Docker registries (like Docker Hub) enable users to query an image by its name and not some SHA representation of it.